.github/workflows/on-release.yml

```diff
@@ -41,6 +41,8 @@ jobs:
       - name: Build and push container images
         uses: docker/build-push-action@v2
         with:
+          build-args: |
+            pkg_version=${{steps.tag.outputs.tag}}
```

Comment:
Super nitpik but mind formatting this like `pkg_version=${{ steps.tag.outputs.tag }}`?